prodtests/full_system_test.sh

```diff
@@ -52,13 +52,22 @@ taskwrapper tofraw.log o2-tof-reco-workflow ${GLOBALDPLOPT} --tof-raw-file-for l
 taskwrapper midraw.log o2-mid-digits-to-raw-workflow ${GLOBALDPLOPT} --mid-raw-outdir raw/MID --mid-raw-perlink  --configKeyValues '"HBFUtils.nHBFPerTF=128;HBFUtils.orbitFirst=0"'
 cat raw/*/*.cfg > rawAll.cfg
 
-ARGS_ALL="--session default"
-taskwrapper reco.log "o2-raw-file-reader-workflow $ARGS_ALL --configKeyValues \"HBFUtils.nHBFPerTF=128\" --delay $TFDELAY --loop $NTIMEFRAMES --max-tf 0 --input-conf rawAll.cfg |  
+# We run the workflow in both CPU-only and With-GPU mode
+for STAGE in "NOGPU" "WITHGPU"; do
+
+  if [[ "$STAGE" = "WITHGPU" ]]; then
```

Comment:
You mixed up forceMemoryPoolSize and forceHostMemoryPoolSize. Could you put for the GPU:
```
TPC_GPU_OPT="GPU_proc.deviceNum=0;GPU_global.deviceType=CUDA;GPU_proc.forceMemoryPoolSize=6000000000;GPU_proc.forceHostMemoryPoolSize=3000000000"
```
and for the CPU
```
TPC_GPU_OPT="GPU_proc.forceHostMemoryPoolSize=${TPCTRACKERSCRATCHMEMORY}"
```

Reply:
Yeah ... this confused me slightly indeed.

Reply:
Sorry, still not correct. Also the GPU part should have the forceHostMemoryPoolSize but we can live with a smaller value. 3 GB should be enough :)